src/lint/collect-spelling-diagnostics.ts

```diff
@@ -59,7 +59,12 @@ let matchers = [
   {
     pattern: /(?<=\S)  +(?! |<\/t(d|h)>)/gu,
     message: 'multiple consecutive spaces are not allowed',
-  }
+  },
+  {
+    pattern: /[Cc]lauses? \d/gu,
```

Comment:
We could use a lookbehind here so that the error points to the number, which is the real issue.
```suggestion
    pattern: /(?<=[Cc]lauses? )\d+/gu,
```

Reply:
Done.